### PR TITLE
Doc new debug methods

### LINKED ISSUE #1342: [Content] Document `debug_getRawReceipts`
### Describe the issue

Document new RPC method `debug_getRawReceipts`. See https://github.com/hyperledger/besu/pull/5476 for more info.

### What page has the issue?

RPC API reference

### Issue type

Missing content

### Change suggestion

_No response_

### More context

_No response_

### Besu documentation issue guidelines

- [X] I read and followed this project's documentation guidelines.

### LINKED ISSUE #1364: Document `debug_getRawTransactions`
Document `debug_getRawTransactions` API method. See https://github.com/hyperledger/besu/pull/5635 for more info.

### PR DESCRIPTION
**Pull request by alexandratran**
Document `debug_getRawTransaction` and `debug_getRawReceipts`.

Preview: https://besu-docs-40jpyeiid-hyperledger.vercel.app/development/public-networks/reference/api#debug_getrawreceipts

fixes #1342 
fixes #1364

### CODE REVIEW

File: docs/public-networks/reference/api/index.md [hunk 1]
```diff
@@ -976,6 +976,85 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"debug_getRawHeader","params":["0
...
+```
+
+# JSON result
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "result": "0x7ef9010c0182fa0db9010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c08353766d"
+}
```
Comment (macfarla) [inline docs/public-networks/reference/api/index.md:1014]: the result here will be an array eg "result" : [ "0xf901a60182c70eb9010000000000000000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000002000000000000000000000008000000000000000000000000000000000040000000001000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000100000000000000400000000000000000000000000000000000000000000000000000000000000000000000000000002000000000100000000000000000000000800000000000000000000000000000000000000000000000000000000000000000000020000000000000000f89df89b947753cfad258efbc52a9a1452e42ffbce9be486cbf863a0ddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3efa00000000000000000000000000828d0386c1122e565f07dd28c7d1340ed5b3315a000000000000000000000000021849e99c31e3113a489d7eb0fd4d8c0edbe47afa00000000000000000000000000000000000000000000000000000000029b92700", "0xf901a70183018e1cb9010000000000000000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000002000000000000000000000008000000000000000000000000000000000040000000001000000000000000000000000000000000000000000000000010000000000000000000000000000000008000000000000000100000000000000000000000000000000000000000000000000000000000000000000000000000000000000200000002000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000020000000000000000f89df89b947753cfad258efbc52a9a1452e42ffbce9be486cbf863a0ddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3efa00000000000000000000000000828d0386c1122e565f07dd28c7d1340ed5b3315a000000000000000000000000069cda9d6cc6ce05982d0b4fdf9480f2991f39b5aa00000000000000000000000000000000000000000000000000000000029b92700" ]

File: docs/public-networks/reference/api/index.md [hunk 2]
```diff
@@ -976,6 +976,85 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"debug_getRawHeader","params":["0
...
+#### Returns
+
+`result`: _object_ - RLP-encoded [transaction object](objects.md#transaction-object)
+
+<!--tabs-->
+
+# curl HTTP request
+
+```bash
+curl -X POST --data '{"jsonrpc":"2.0","method":"debug_getRawBlock","params":["0x3a2fd1a5ea9ffee477f449be53a49398533d2c006a5815023920d1c397298df3"],"id":1}' http://127.0.0.1:8545
```
Comment (macfarla) [inline docs/public-networks/reference/api/index.md:1037]:
```suggestion
curl -X POST --data '{"jsonrpc":"2.0","method":"debug_getRawTransaction","params":["0x3a2fd1a5ea9ffee477f449be53a49398533d2c006a5815023920d1c397298df3"],"id":1}' http://127.0.0.1:8545
```

File: docs/public-networks/reference/api/index.md [hunk 3]
```diff
@@ -976,6 +976,85 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"debug_getRawHeader","params":["0
...
+```
+
+# wscat WS request
+
+```bash
+{"jsonrpc":"2.0","method":"debug_getRawBlock","params":["0x3a2fd1a5ea9ffee477f449be53a49398533d2c006a5815023920d1c397298df3"],"id":1}
```
Comment (macfarla) [inline docs/public-networks/reference/api/index.md:1043]:
```suggestion
{"jsonrpc":"2.0","method":"debug_getRawTransaction","params":["0x3a2fd1a5ea9ffee477f449be53a49398533d2c006a5815023920d1c397298df3"],"id":1}
```

File: docs/public-networks/reference/api/index.md [hunk 4]
```diff
@@ -976,6 +976,85 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"debug_getRawHeader","params":["0
...
+```
+
+# JSON result
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "result": "0xe6808609184e72a0008303000094b0920c523d582040f2bcb1bd7fb1c7c1ecebdb3480801c8080"
```
Comment (gfukushima) [inline docs/public-networks/reference/api/index.md:1052]:
```suggestion
  "result": "0xf8678084342770c182520894658bdf435d810c91414ec09147daa6db624063798203e880820a95a0af5fc351b9e457a31f37c84e5cd99dd3c5de60af3de33c6f4160177a2c786a60a0201da7a21046af55837330a2c52fc1543cd4d9ead00ddf178dd96935b607ff9b"
```

Comment (gfukushima) [reply docs/public-networks/reference/api/index.md:1052]: this would be a more realistic result

**Review by macfarla — APPROVED** [review]

File: docs/public-networks/reference/api/index.md [hunk 5]
```diff
@@ -976,6 +976,88 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"debug_getRawHeader","params":["0
...
 <!--/tabs-->
 
+### `debug_getRawReceipts`
+
+Returns the [RLP encoding](https://ethereum.org/en/developers/docs/data-structures-and-encoding/rlp/)
+of the transaction receipts of the specified block.
+
+#### Parameters
+
+`blockNumber`: _string_ - integer representing a block number or one of the string tags `latest`,
```
Comment (bgravenorst) [inline docs/public-networks/reference/api/index.md:986]: This applies throughout for the `blockNumber` parameter. Should we say, "hexadecimal integer representing..."?
Also, should we mention `finalized` and `safe` as possible options?

Could be implemented in separate PR if you agree. Would be a "good first issue".

**Review by bgravenorst — APPROVED** [review]
It may be easier to implement my suggestion in a separate PR (if you agree with it)

File: docs/public-networks/reference/api/index.md (hunk 5 above)
Comment (alexandratran) [reply docs/public-networks/reference/api/index.md:986]: Raised issue #1385

**Pull request merged**